burr/core/application.py

```diff
@@ -103,6 +103,31 @@ def _adjust_single_step_output(
     _raise_fn_return_validation_error(output, action_name)
 
 
+def _remap_context_variable(run_method: Callable, inputs: Dict[str, Any]) -> dict:
```

Comment:
We should do this with any `__` variable... `__tracer` will hit the same problem. Could hardcode this, but we should consider being more aware of the context.